networkit/cpp/community/LouvainMapEquation.cpp

```diff
@@ -34,6 +34,21 @@ LouvainMapEquation::LouvainMapEquation(const Graph &graph, bool hierarchical, co
     result = Partition(graph.upperNodeIdBound());
 }
 
+LouvainMapEquation::LouvainMapEquation(const Graph &graph, bool hierarchical, count maxIterations,
+                                       std::string_view parallelizationStrategy)
+    : CommunityDetectionAlgorithm(graph), hierarchical(hierarchical), maxIterations(maxIterations),
+      clusterCut(graph.upperNodeIdBound()), clusterVolume(graph.upperNodeIdBound()) {
+    parallelizationType = convertStringToParallelizationType(parallelizationStrategy);
```

Comment:
this line should be part of the initializer list and not in the function body

Reply:
The other variables in this function could be moved into the initializer list as well